### DISCUSSION TITLE
Can I find the permeability of a porous medium a with Porespy ?

### DISCUSSION THREAD

**Discussion by josephinoo** [PMEAL/porespy#769] [Q&A]
Hi, I am an undergraduate in computer science and I am doing a research on permeability and understanding darcy's law.

**Comment by ma-sadeghi** [comment]
Hi, take a look at his example: https://openpnm.org/examples/applications/absolute_permeability.html

**Reply by jgostick** [reply]
So, no, you can't do it with porespy, but you can extract a network with porespy then import that network into openpnm and compute the permeability of the pore network.
